gateway/src/apicast/policy/cors/apicast-policy.json

```diff
@@ -46,7 +46,7 @@
         }
       },
       "allow_origin": {
-        "description": "Origin allowed for CORS requests. The field expects only one origin (e.g. https://example.com) or '*'. If left blank, the value of the 'Origin' request header will be used.",
+        "description": "Origin allowed for CORS requests. The field expects only one origin (e.g. 'https://example.com') or '*'. If left blank, the value of the 'Origin' request header will be used. In order to allow more than one origin it is possible to use a regular expression, if it matches with Origin header value, the value will be set to the Origin Value. In case it does not match, the header will not set at all. Example: '(api|web).test.com' wil match both 'api.test.com' and 'web.test.com'.",
```

Comment:
```suggestion
        "description": "Origin allowed for CORS requests. The field expects only one origin (e.g. 'https://example.com') or '*'. If left blank, the value of the 'Origin' request header will be used. To allow multiple origins, you can use a regular expression. if the regular expression matches with the `Origin` header value, the value will be set to the origin value. In case it does not match, the header will not br set. Example: '(api|web).test.com' matches both 'api.test.com' and 'web.test.com'.",
```